chromium_src/third_party/blink/renderer/bindings/modules/v8/v8_storage_estimate.h

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_THIRD_PARTY_BLINK_RENDERER_BINDINGS_MODULES_V8_V8_STORAGE_ESTIMATE_H_
+#define BRAVE_CHROMIUM_SRC_THIRD_PARTY_BLINK_RENDERER_BINDINGS_MODULES_V8_V8_STORAGE_ESTIMATE_H_
+
+#define StorageEstimate StorageEstimate_ChromiumImpl
+#include "../gen/third_party/blink/renderer/bindings/modules/v8/v8_storage_estimate.h"  // IWYU pragma: export
+#undef StorageEstimate
+
+namespace blink {
+
+class StorageEstimate : public StorageEstimate_ChromiumImpl {
+ public:
+  static StorageEstimate* Create() {
+    return MakeGarbageCollected<StorageEstimate>();
+  }
+
+  using StorageEstimate_ChromiumImpl::StorageEstimate_ChromiumImpl;
+
+  void setQuota(uint64_t quota) {
```

Comment:
does anyone know what the original purpose of this was? It would be helpful if that was captured here

Reply:
https://github.com/brave/brave-browser/issues/12370

Reply:
can we add a link/comment here since this is going to be hard to track with git blame?

Reply:
Added a comment in 5f260d9a9ba